### PR TITLE
Ccd 1737 Kustomize admin-web

### PR DESCRIPTION
**Pull request by nathan-clark**
https://tools.hmcts.net/jira/browse/CCD-1737

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by MSancaktutar — DISMISSED** [review]
lgtm

**Review by MSancaktutar — DISMISSED** [review]
LGTM

File: k8s/namespaces/ccd/ccd-admin-web/aat.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccd-admin-web
+spec:
+  releaseName: ccd-admin-web
+  values:
+    nodejs:
+      environment:
+        ADMINWEB_ELASTIC_INDEX_URL: http://ccd-definition-store-api-aat.service.core-compute-aat.internal/elastic-support/index
```
Comment (smathangi) [inline k8s/namespaces/ccd/ccd-admin-web/aat.yaml:10]: you might need to add this to aat `kustomization.yaml` as well  (under patchesStrategicMerge)

Comment (timja) [reply k8s/namespaces/ccd/ccd-admin-web/aat.yaml:10]: Well, you're right that if you want aat.yaml to be loaded this needs adding.

But this is just setting it to the same value that's in the chart:
https://github.com/hmcts/ccd-admin-web/blob/ab7b4bf287a802657818a44c9be16e44e533f2bb/charts/ccd-admin-web/values.yaml#L48

This file should be removed or just have a dummy var if you want to be able to easily update env values, and if you keep the file you need to add it to `patchesStrategicMerge`

File: k8s/namespaces/ccd/ccd-admin-web/ccd-admin-web.yaml [hunk 2]
```diff
@@ -21,7 +13,3 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ccd/admin-web:prod-ab7b4bf-20210810173243
```
Comment (timja) [inline k8s/namespaces/ccd/ccd-admin-web/ccd-admin-web.yaml:15]:
```suggestion
      image: hmctspublic.azurecr.io/ccd/admin-web:prod-ab7b4bf-20210810173243  #{"$imagepolicy": "flux-system:ccd-admin-web"}
```

**Review by timja — CHANGES_REQUESTED** [review]

File: k8s/namespaces/ccd/ccd-admin-web/aat.yaml (hunk 1 above)
Comment (nathan-clark) [reply k8s/namespaces/ccd/ccd-admin-web/aat.yaml:10]: Was keeping it simple, ok to delete @MSancaktutar ?

**Review by MSancaktutar — DISMISSED** [review]

**Comment by timja** [comment]
@MSancaktutar https://github.com/hmcts/cnp-flux-config/pull/10928/files#r692025691

also conflicts still need resolving

File: k8s/namespaces/ccd/ccd-admin-web/aat.yaml (hunk 1 above)
Comment (MSancaktutar) [reply k8s/namespaces/ccd/ccd-admin-web/aat.yaml]: Deleted the file.

**Review by MSancaktutar — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**